app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheet.kt

```diff
@@ -93,8 +98,11 @@ class BloodPressureEntrySheet : BottomSheetActivity() {
     diastolicEditText.requestFocus()
   }
 
-  fun finishAndScheduleAppointment(patientUuid: UUID) {
-    startActivity(ScheduleAppointmentSheet.intent(this, patientUuid))
+  fun finishAfterBloodPressureSaved() {
```

Comment:
Hey… this function name is no longer relevant right? It now finishes immediately.